systemtest/src/test/java/io/strimzi/systemtest/specific/HelmChartST.java

```diff
@@ -2,9 +2,10 @@
  * Copyright Strimzi authors.
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
-package io.strimzi.systemtest;
+package io.strimzi.systemtest.specific;
```

Comment:
Perhaps this should be `infrastructure` or `deployment` or `install`

Reply:
I think we will remove this class soon. I want to do same thing as I did with OLM installation - add a posibility to run most of the tests against CO intaller via helm

Reply:
Makes sense to me!